EIPS/eip-pbmrc1.md

```diff
@@ -10,64 +10,84 @@ created: 2023-04-01
 requires: 165, 173, 1155
 ---
 
-<!-- Notes: replace PRBMRC with an EIP number upon creating a PR to EIP Main repo -->
+<!-- Notes: replace PRBMRC with EIP upon creating a PR to EIP Main repo -->
 ## Abstract
 
-This PBMRC outlines a smart contract interface that builds upon the [ERC-1155](./eip-1155.md) standard to introduce the concept of a purpose bound money (PBM) defined in the [Project Orchid Whitepaper](../assets/eip-pbmrc1/MAS-Project-Orchid.pdf) 
-
-It builts upon the [ERC-1155](./eip-1155.md) standard to leverage upon existing widespread support that wallet providers has implemeneted to display the PBM and to trigger various transfer logic.
+This PBMRC outlines a smart contract interface that builds upon the [ERC-1155](./eip-1155.md) standard to introduce the concept of a purpose bound money (PBM) defined in the [Project Orchid Whitepaper](../assets/eip-pbmrc1/MAS-Project-Orchid.pdf).
 
+It builds upon the [ERC-1155](./eip-1155.md) standard, by leveraging pre-existing, widespread support that wallet providers have implemented, to display the PBM and trigger various transfer logic.
 
 ## Motivation
 
-Purpose Bound Money (PBM) refers to a protocol that specifies the conditions under which an underlying digital token of value can be used. PBMs can be a bearer or order instruments with self-contained programming logic, and can be transferred between two parties without intermediaries. 
+The establishment of this protocol seeks to forestalls technology fragmentation and consequently a lack of interoperability. By making the PBM specification open, it gives new participants easy and free access to the pre-existing market standards, enabling interoperability across different platforms, wallets, payment systems and rails. This would lower cost of entry for new participants, foster a vibrant payment landscape and prevent the development of walled gardens and monopolies, ultimately leading to more efficient, affordable services and better user experiences.
+
+## Definitions
```

Comment:
Wah i like this definition section <3 <3 <3 <3


![image](https://github.com/StraitsX/EIPs/assets/1174679/b91eed39-2688-4cce-851c-a39175f45154)